index.html

```diff
@@ -0,0 +1,126 @@
+      <p>Size: big </p>
+      <p>Weight: heavy</p>
+      <footer>
+        <h6>Specifications valid until 4/17/16</h6>
+      </footer>
+    </section>
+    <section class="sec3">
+      <header id="header" class="">
+        <h3>Pricing</h3>
+      </header>
+      <p>1: $9.99</p>
+      <p>2-50: $6.99</p>
+      <p>51-100: $3.99</p>
+    </section>
+  </article>
+
+  <article class="card">
+    <header id="header" class="">
+      <h1>Red Squibby</h1>
+    </header>
+    <section class="sec1">
+      <img src="red.png" class="pic" alt="">
+      <p> Ker ima Lorem Ipsum relativno enakomerno razporeditev znakov uspešno nadomesti začasna vsebinsko pomenska besedila. </p>
+      <div class="availability">Available</div>
+    </section>
+    <section class="sec2">
+      <header id="header" class="">
+        <h4>Specifications</h4>
+      </header>
+      <p>Size: big</p>
+      <p>Weight: heavy</p>
+      <footer>
+        <h6>Specifications valid until 4/17/16</h6>
+      </footer>
+    </section>
+    <section class="sec3">
+      <header id="header" class="">
+        <h3>Pricing</h3>
+      </header>
+      <p>1: $9.99</p>
+      <p>2-50: $6.99</p>
+      <p>51-100: $3.99</p>
+    </section>
+  </article>
+
+  <article class="card">
+    <header id="header" class="">
+      <h1>Blue Squibby</h1>
+    </header>
+    <section class="sec1">
+      <img src="blue.png" class="pic" alt="">
+      <p>Veliko namizno založniških programov in spletnih urejevalnikov uporablja Lorem Ipsum kot privzeti slepi tekst.</p>
+      <div class="availability">Available</div>
+    </section>
+    <section class="sec2">
+      <header id="header" class="">
+        <h4>Specifications</h4>
+      </header>
+      <p>Size: big</p>
+      <p>Weight: heavy</p>
+      <footer>
+        <h6>Specifications valid until 4/17/16</h6>
+      </footer>
+    </section>
+    <section class="sec3">
+      <header id="header" class="">
+        <h3>Pricing</h3>
+      </header>
+      <p>1: $9.99</p>
+      <p>2-50: $6.99</p>
+      <p>51-100: $3.99</p>
+    </section>
+  </article>
+
+  <article class="card">
+    <header id="header" class="">
+      <h1>Pink Squibby</h1>
+    </header>
+    <section class="sec1">
+      <img src="pink.png" class="pic" alt="">
+      <p>Tekom let so namreč nastale številne različice tega slepega teksta, bodisi nenačrtovano ali namenoma, z različnimi šaljivimi in drugimi vložki.</p>
+      <div class="availability">Available</div>
+    </section>
+    <section class="sec2">
+      <header id="header" class="">
+        <h4>Specifications</h4>
+      </header>
+      <p>Size: big</p>
+      <p>Weight: heavy</p>
+      <footer>
+        <h6>Specifications valid until 4/17/16</h6>
+      </footer>
+    </section>
+    <section class="sec3">
+      <header id="header" class="">
+        <h3>Pricing</h3>
+      </header>
+      <p>1: $9.99</p>
+      <p>2-50: $6.99</p>
+      <p>51-100: $3.99</p>
+    </section>
+  </article>
+</body>
+</html>
```

Comment:
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior